packages/core/src/schemas/publisherJsonMetadata.json

```diff
@@ -50,10 +50,12 @@
                   "type": "string"
                 },
                 "website": {
-                  "type": "string"
+                  "type": "string",
+                  "pattern": "^(https?://[^/]+)[/]{0,1}"
```

Comment:
If I read this correctly, `[^/]+)[/]{0,1}` says "one or more non-forward slash characters, followed by an optional forward slash". Since we don't validate the entire URI, nor do we match through end of string, isn't that the same as just saying "at least one non-forward slash character"?

Reply:
I'm really not sure. These were generated by ChatGPT. I tried it with a few "bad" strings and it seemed to only allow the right thing, but I can stress test harder.

Reply:
Not saying it won't work, just that this regex isn't as succinct as it could be. For future readers of this code, they could be left wondering what was really intended by this.

I'd probably just change this to `^https://` and be done with it.